docs/scope.md

```diff
@@ -0,0 +1,9 @@
+### Scope of this repository
+
+Both javaagent and library-based approaches to the following:
+
+* Instrumentation for specific Java libraries and frameworks
+  * Emitting spans and metrics (and in the future logs)
+* System metrics 
+* Logging integrations
+* Spring integrations
```

Comment:
```suggestion
* Framework integrations (e.g., Spring)
```

Honestly nothing comes to mind but there must be more frameworks than just Spring right? :D

Reply:
E.g. Ratpack, Play :)

Reply:
I think frameworks are covered under Instrumentation bullet above. Narrowed this down to Spring Boot starters for now(?)